jobs/index.js

```diff
@@ -0,0 +1,22 @@
+const express = require('express');
+const cors = require('cors');
+const axios = require('axios');
+
+const app = express();
+app.use(express.json());
+app.use(cors());
+
+app.get('/api/jobs/github', cors(), async (req, res) => {
```

Comment:
@jaylee20  has picked up the ticket for this and will be connecting to the API, storing the data and serving to the front-end.